config.php.dist

```diff
@@ -69,6 +69,16 @@ return [
         'contact@example.com',
     ],
 
+     /*
+    |--------------------------------------------------------------------------
+    | Default message
+    |--------------------------------------------------------------------------
+    */
+
+    'default-message' => [
```

Comment:
Thanks for your contribution. We need just a little naming improvement. That's not email body (as name 'default-message' might suggest) but this is it's title – so it should be something like 'default_title' or 'default_message_title'. 

And second thing, to stay consistent with whole library please use _ instead of -. So in your example default_message and not default-message.

Reply:
Thanks for feedback, I just pushed fixes for that.